frontend/src/components/App/App.js

```diff
@@ -1,25 +1,80 @@
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
-import { SignInPage, SignUpPage, SignedIn, SignedUp } from '../';
+import { BrowserRouter, Route, NavLink } from 'react-router-dom';
+import { SignInPage, SignUpPage } from '../';
+
+import { Switch, Redirect } from 'react-router-dom';
+import { Provider, connect } from 'react-redux';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
```

Comment:
Should only be using combineReducers in /store/reducers/index.js. Need to import reducer from index.js and use that for createStore() something like 
`import rootReducer from '../../store/reducers';
let store = createStore(rootReducer, applyMiddleware(thunk));`